django/utils/feedgenerator.py

```diff
@@ -187,7 +188,10 @@ class RssFeed(SyndicationFeed):
     content_type = 'application/rss+xml; charset=utf-8'
 
     def write(self, outfile, encoding):
-        handler = SimplerXMLGenerator(outfile, encoding)
+        if sys.version_info.major == 3 and sys.version_info.minor >= 2:
```

Comment:
The minimum supported python version for the next Django release is 3.8 so this version check is not needed.